tailwind.config.js

```diff
@@ -1,5 +1,4 @@
 module.exports = {
-  mode: 'jit',
```

Comment:
Are you sure jit is not used by any file?

Reply:
I have removed it because I have found an issue in MAC and I have reported the issue in the group. But I have forgotten to add it back when I pushed for this PR. 

Added it back